src/Components/MapboxMap/controls/AhtiNavigationControl.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { NavigationControl } from 'react-map-gl';
+
+export default class AhtiNavigationControl extends NavigationControl {
+  _render() {
+    const { className, showZoom, zoomInLabel, zoomOutLabel } = this.props;
+
+    return (
+      <div
+        className={`mapboxgl-ctrl mapboxgl-ctrl-group ${className}`}
+        ref={this._containerRef}
+      >
+        {showZoom &&
+          this._renderButton(
+            'zoom-in',
+            zoomInLabel ? zoomInLabel : 'Zoom In',
```

Comment:
The reason these two extensions are added (NavigationControl and GeolocateControl) is to have a quick fix for the lack of ability to customize these texts and to fix a bug in the geolocate control. The plan is to submit a fix to the `react-map-gl` project once there is time to complete it.

In addition, the reason for using js for these instead of ts is that the type definitions for these controls don't allow a clean way to extend props.

Reply:
So the reason why the translation is not done in the component directly is because we would like to eventually get rid of it completely and rely on the library to provide the functionality we need.

Reply:
Fair enough!